test/server/admin/stats_handler_test.cc

```diff
@@ -117,11 +117,6 @@ class StatsHandlerTest {
     }
   }
 
-  void setRegexType(Regex::Type type) {
-    scoped_runtime_.mergeValues({{"envoy.reloadable_features.admin_stats_filter_use_re2",
```

Comment:
since no longer using the scoped runtime can we remove the include and the dependency on the test_runtime_lib in build files?

Reply:
Done